x-pack/plugins/kubernetes_security/common/constants.ts

```diff
@@ -41,3 +39,19 @@ export const COUNT_WIDGET_KEY_NAMESPACE = 'CountNamespaceWidgets';
 export const COUNT_WIDGET_KEY_NODES = 'CountNodesWidgets';
 export const COUNT_WIDGET_KEY_PODS = 'CountPodsWidgets';
 export const COUNT_WIDGET_KEY_CONTAINER_IMAGES = 'CountContainerImagesWidgets';
+
+export const DEFAULT_QUERY = '{"bool":{"must":[],"filter":[],"should":[],"must_not":[]}}';
+export const DEFAULT_FILTER_QUERY =
```

Comment:
wondering if we should take the opportunity to let it as an object instead? the same `DEFAULT_FILTER` below, then we can avoid having to call `JSON.stringify` whenever modifying, and only call it when fetching?

Reply:
Yea I was thinking about refactoring all the `JSON.parse` and `JSON.stringify` as well. I will do it in a separate PR since it will involve the `aggregate` and `count` routes